inc/class-admin.php

```diff
@@ -193,8 +193,8 @@ public function list_row_action( array $actions, WP_Post $post ) {
 	public function clone_post() {
```

Comment:
This always runs on `admin_init`, not just when cloning the posts. Thus, it causes problems.

Reply:
It seems like regardless of this change, line 205 is going to prevent anything from happening since `$nonce` is not defined. Looks like this came from one of the latest commits fixing linting issues.

We should check there for the value of `$_GET['clone_post']` on line 205 along with the `$pid`.

@peterwilsoncc could you update that in this PR? Then I'll give it a 👍 

Thanks man!